### PR TITLE
Update manifest.py for demo

### PR DESCRIPTION
**Pull request by Railia**
File has been updated to get Live preview of module

### CODE REVIEW

File: theme_kit/__manifest__.py [hunk 1]
```diff
@@ -38,4 +39,15 @@
 
     "auto_install": False,
     "installable": True,
+    
+    "demo_title": "Brand kit",
+    "demo_addons": [
```
Comment (ilmir-k) [inline theme_kit/__manifest__.py]: Add web_debranding and web_login_background into demo_addons to be able to see it on demo page

**Comment by ilmir-k** [comment]
@Railia 
don't forget to add `live_test_url` parameter too

File: theme_kit/__manifest__.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
     "name": """Brand kit""",
     "summary": """Brand your odoo instance in few clicks""",
     "category": "Debranding",
+    "live_test_url": "http://apps.it-projects.info/shop/product/theme_kit?version=11.0",
```
Comment (ilmir-k) [inline theme_kit/__manifest__.py]: @Railia incorrect url, you should use value from "demo url", not `theme_kit`

File: theme_kit/__manifest__.py [hunk 3]
```diff
@@ -38,4 +40,16 @@
 
     "auto_install": False,
     "installable": True,
+    
+    "demo_title": "Brand kit",
+    "demo_addons": [
+        "web_debranding", "web_login_background"
```
Comment (ilmir-k) [inline theme_kit/__manifest__.py]: it is better to use new line per each module like this:
```
"demo_addons": [
            "web_debranding", 
            "web_login_background",
],
```

**Review by ilmir-k — APPROVED** [review]

File: theme_kit/__manifest__.py [hunk 4]
```diff
@@ -30,6 +31,7 @@
     "qweb": [
     ],
     "demo": [
+         # "theme_data.xml",
```
Comment (yelizariev) [inline theme_kit/__manifest__.py]: No need for this

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by ilmir-k — APPROVED** [review]

**Comment by ilmir-k** [comment]
@yelizariev your turn

**Pull request merged**